src/dash-table/components/Button/ExportButton/index.tsx

```diff
@@ -0,0 +1,46 @@
+import XLSX from 'xlsx';
```

Comment:
Structurally this should be in a `components/Export` folder instead of the more explicitly named `Button/ExportButton` folder -- the fact that it's implemented with a button is of no importance / is reminiscent of Hungarian notation and is potentially misleading.